internal/modulespecifiers/types.go

```diff
@@ -9,14 +9,6 @@ import (
 	"github.com/microsoft/typescript-go/internal/tspath"
 )
 
-type SourceFileForSpecifierGeneration interface {
```

Comment:
These interfaces (this and `HasFileName`) appear to do nothing. I could theoretically move the metadata calls onto these and keep it working, but I don't know if it's actually worth it right now.